app/controllers/v0/pension_claims_controller.rb

```diff
@@ -12,40 +12,31 @@ def claim_class
       SavedClaim::Pension
     end
 
-    # rubocop:disable Metrics/MethodLength
     def show
       claim = claim_class.find_by!({ guid: params[:id] }) # will raise ActiveRecord::NotFound
-
       form_submission = claim.form_submissions&.order(id: :asc)&.last
       submission_attempt = form_submission&.form_submission_attempts&.order(created_at: :asc)&.last
-
       if submission_attempt
         # this is to satisfy frontend check for successful submission
         state = submission_attempt.aasm_state == 'failure' ? 'failure' : 'success'
-        response = {
-          data: {
-            id: claim.id,
-            form_id: claim.form_id,
-            guid: claim.guid,
-            attributes: {
-              state:,
-              benefits_intake_uuid: form_submission.benefits_intake_uuid,
-              form_type: form_submission.form_type,
-              attempt_id: submission_attempt.id,
-              aasm_state: submission_attempt.aasm_state
-            }
-          }
-        }
+        response = format_show_response(claim, state, form_submission, submission_attempt)
       end
-
       render(json: response)
+    rescue ActiveRecord::RecordNotFound => e
```

Comment:
Does Ruby does require some form of “begin” to appear before the rescue?

Reply:
A method 'def' can serve as a begin - so in this case, the rescue looks for any non-caught errors inside the method

Reply:
Thank you for explaining it.